flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/InputGateConcurrentTest.java

```diff
@@ -88,7 +90,9 @@ public void testConsumptionWithRemoteChannels() throws Exception {
 		final ConnectionManager connManager = createDummyConnectionManager();
 		final Source[] sources = new Source[numberOfChannels];
 
-		final SingleInputGate gate = createSingleInputGate(numberOfChannels);
```

Comment:
True, actually I do not mind short-hands. Though, I would remove one of short-hands with 3 parameters. I think builder is more readable in case of many parameters.